var/spack/repos/builtin/packages/py-tensorboard/package.py

```diff
@@ -14,29 +16,29 @@ class PyTensorboard(Package):
 
     maintainers = ['aweits']
 
+    version('2.4.1', sha256='736dc204aa292d221f5871077e60994a9a9ea8e33b841f0d754d510fe6cc7635')
+    version('2.4.0', sha256='28a30794c1c797357b2086477394b59afa0b18ca48592ca3c0627f7f10536373')
     version('2.3.0', sha256='947a58702c2841eb4559637dbf8639633f79de9a0f422be9737f3563a1725440')
     version('2.2.0', sha256='d0dfbf0e4b3b5ebbc3fafa6d281d4b9aa5478eac6bac3330652ab6674278ab77')
 
     depends_on('python@2.7:2.8,3.2:', type=('build', 'run'))
-    depends_on('bazel@0.26.1:', type='build')
+    depends_on('bazel@2.1.0:', type='build', when='@2.2.0:')
     depends_on('py-setuptools@41.0.0:', type=('build', 'run'))
     depends_on('py-absl-py@0.4:', type=('build', 'run'))
     depends_on('py-markdown@2.6.8:', type=('build', 'run'))
     depends_on('py-requests@2.21.0:2.999', type=('build', 'run'))
     depends_on('py-futures@3.1.1:', type=('build', 'run'), when='^python@:2')
-    depends_on('py-grpcio@1.24.3:', type=('build', 'run'), when='@2.3.0')
-    depends_on('py-grpcio@1.23.3:', type=('build', 'run'), when='@2.2.0')
+    depends_on('py-grpcio@1.24.3:', type=('build', 'run'), when='@2.3:')
+    depends_on('py-grpcio@1.23.3:', type=('build', 'run'), when='@2.2')
     depends_on('py-google-auth@1.6.3:1.99.99', type=('build', 'run'))
     depends_on('py-numpy@1.12.0:', type=('build', 'run'))
     depends_on('py-protobuf@3.6.0:', type=('build', 'run'))
     depends_on('py-six@1.10.0:', type=('build', 'run'))
     depends_on('py-werkzeug@0.11.15:', type=('build', 'run'))
-    depends_on('py-wheel', type='build')
-    depends_on('py-wheel@0.26:', type='build', when='@0.6: ^python@3:')
+    depends_on('py-wheel', type=('build', 'run'))
+    depends_on('py-wheel@0.26:', type=('build', 'run'), when='@0.6: ^python@3:')
```

Comment:
Why is wheel needed at run-time?

Reply:
the short answer is that if you run "tensorboard" without that dep, you get this output:

```shell
[aweits@localhost ~]$ tensorboard --logdir /tmp/foo
Traceback (most recent call last):
<some bits removed for brevity>
  File "/home/aweits/GIT/UPSTREAMSPACK/spack/opt/spack/linux-centos8-skylake_avx512/gcc-9.3.0/py-setuptools-50.3.2-vu5bjaj73n63hh3xacems2o5vmhqt66r/lib/python3.8/site-packages/pkg_resources/__init__.py", line 770, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'wheel>=0.26' distribution was not found and is required by tensorboard
spack:/home/aweits/GIT/UPSTREAMSPACK/spack
```